src/Microsoft.Android.Sdk.ILLink/SetupStep.cs

```diff
@@ -43,9 +46,13 @@ protected override void Process ()
 			subSteps2.Add (new PreserveRegistrations (cache));
 			subSteps2.Add (new PreserveJavaInterfaces ());
 
-			InsertAfter (new FixAbstractMethodsStep (cache), "RemoveUnreachableBlocksStep");
-			InsertAfter (subSteps2, "RemoveUnreachableBlocksStep");
-			InsertAfter (subSteps1, "RemoveUnreachableBlocksStep");
+			InsertAfter (new FixAbstractMethodsStep (cache), "SetupStep");
+			InsertAfter (subSteps2, "SetupStep");
+			InsertAfter (subSteps1, "SetupStep");
+
+			// temporary workaround: this call forces illink to process all the assemblies
+			foreach (var assembly in (IEnumerable<AssemblyDefinition>)getReferencedAssembliesMethod.Invoke (Context, null))
+				Context.LogMessage ($"Reference assembly to process: {assembly}");
```

Comment:
Is this something we can make use of https://github.com/xamarin/xamarin-android/pull/5317/files#diff-be3e7a80e4a796e2a4af1bbb778767f3f7d9c431534156e771e4ff7dc885e5c5R12 to do? (once its merged)

Reply:
yes, we can use this call to replace `GetAssembliesStep`. it wouldn't live long though, the workaround will be most likely around just for few weeks till we have new API available.